week-3/A-array-find/exercise.js

```diff
@@ -6,9 +6,14 @@
 // write your code here
 
 var names = ["Rakesh", "Alexandra", "Annam", "Mikey", "Karim", "Ahmed"];
-var longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+var longNameThatStartsWithA = names.find(findLongNameThatStartsWithA);
+
+function findLongNameThatStartsWithA(names) {
+  return names.length >= 7 && names[0] === "A";
```

Comment:
Great stuff! Although the instructions say "longer than" 7 - but that's a minor point :)